### ISSUE TITLE
Dashboard getChartData method fails when Elastic settings not available

### ISSUE THREAD

**Issue by brylie** [apinf/api-umbrella-dashboard#927]
While in the process of re-working our settings handling, we get the following error when accessing the dashboard without Elastic settings:

```
Exception while invoking method 'getChartData' TypeError: Cannot read property 'host' of undefined
     at [object Object].Meteor.methods.getChartData (server/methods/elasticSearch.js:36:44)
```

**Comment by brylie** [comment]
@frenchbread we will need to revisit this bug soon. It is affecting @Alapan's work somewhat.

**Comment by frenchbread** [comment]
There are already couple of checks for settings on elastic-rest package for meteor. Or is this bug on apinf's side?

**Comment by brylie** [comment]
I think it is in the Apinf `getChartData` method.

**Comment by Alapan** [comment]
@frenchbread @brylie Is there any update on this issue?

**Comment by bajiat** [comment]
@frenchbread Could you take this also as a sprint task?

**Comment by frenchbread** [comment]
@bajiat Self-assigned

**Comment by bajiat** [comment]
@frenchbread Any update on the progress?

**Comment by frenchbread** [comment]
Can not reproduce this error anymore. Since we depreciated elastic-rest package in #944. PR contains a logic [here](https://github.com/apinf/api-umbrella-dashboard/pull/944/files#diff-a742e565a035ab75b88d8f631ae18db1R15) to check if ES settings are provided. Should we close this issue then?

**Comment by brylie** [comment]
Closing, since resolved by PR #944
